tools/codegen-options/main.go

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Tabs 😱  is that a Go requirement?

Reply:
LOL yup, the language comes with an opinionated format via the gofmt tool, which uses tabs for indention and spaces for various alignment things, e.g. making struct fields line up.